services/docker-compose.devel.yml

```diff
@@ -1,54 +1,48 @@
 # Overrides docker-compose.yml config when deploying services in development mode,
 #  i.e. binding with source code in source
-#
 # NOTES:
 # - port 3000 used for ptsv
 #
 version: "3.8"
+x-common-environment: &common-environment
+  DEBUG : "true"
+  SC_BOOT_MODE : debug
```

Comment:
I had a conversation with @bisgaard-itis about this. what is the concept behind having both DEBUG=true and SC_BOOT_MODE=debug?

Reply:
documented in code